presto-spi/src/main/java/com/facebook/presto/spi/function/SqlInvokedAggregationFunctionImplementation.java

```diff
@@ -59,4 +73,28 @@ public boolean isOrderSensitive()
     {
         return isOrderSensitive;
     }
+
+    @Override
+    public List<Type> getParameterTypes()
+    {
+        return parameterTypes;
+    }
+
+    @Override
+    public AggregationMetadata getAggregationMetadata()
+    {
+        return null;
```

Comment:
I believe it should be safe or even preferable to throw here, since they should be called.